krun/platform.py

```diff
@@ -400,6 +402,7 @@ def check_preliminaries(self):
         self._check_cpu_scaler()
         self._check_perf_samplerate()
         self._check_tickless_kernel()
+        self._check_aslr_disabled()
```

Comment:
The function `check_preliminaries` is called from `krun.py` and is guarded like this:

```
    if not args.develop:                                                        
        platform.check_preliminaries()                                          
    else:                                                                       
        # Needed to skip the use of certain tools and techniques.               
        # E.g. taskset on Linux, and switching user.                            
        warn("Not checking platform prerequisites due to developer mode")       
        platform.developer_mode = True
```